### DISCUSSION TITLE
open file *.fnx on other system error

### DISCUSSION THREAD

**Discussion by karoltomanek** [tsujan/FeatherNotes#352] [Q&A] [answered]
I have LMDE 6 and FeatherNotes 1.1.1 and working ok, but when I open the same file on Garuda ora Manjaro then i cant - error, cannot open the file,  default kernels, actual systems, I try change kernel for older

**Comment by tsujan** [comment, marked as answer]
> I try change kernel for older

The kernel can't be related to such problems. You have to search for another cause, like MIME type handling.

Can the file be opened when you drag and drop it into FeatherNotes?

**Reply by karoltomanek** [reply]
NO!!! i can't.

**Reply by tsujan** [reply]
If we suppose that the installable package is OK, the database may have not been updated successfully. That may be caused by a problem in `shared-mime-info`.

I vaguely remember that a similar problem (not limited to FeatherNotes) happened after a Manjaro upgrade months ago, but it was fixed by another upgrade that came a few hours later.

**Comment by karoltomanek** [comment]
but when I create new file .fnx, close the program, double click on file .fnx then is OK

**Reply by tsujan** [reply]
What does the error message say exactly? Is there an additional message when you use a terminal to open the file?

**Reply by karoltomanek** [reply]
normally and by the terminal is "can't open the file" (from polish), by menu File/Open ... that is the same

**Reply by tsujan** [reply]
Is the message shown in the terminal or only in FeatherNote's warning-bar?

**Reply by karoltomanek** [reply]
only in FeatherNote's warning-bar (bottom, red belt)

**Reply by tsujan** [reply]
That shows that the file *is* given to FeatherNotes, but the app cannot open it.

For now, what comes to my mind is a permission problem, or a corrupted file, or a corrupted/incompatible binary/package — unless there is more info you haven't found or mentioned.

If exactly the same file can be opened without problem in your first system, then we could rule out file corruption.

**Reply by karoltomanek** [reply]
I uninstall the soft and compile from git, now IS OK - WORKING EVERYTHING, so.... arch repo has problem? maybe, like with instal new kernels under endeavour and today width manjaro -- no new boot menu (fixed) but....
under LMDE 6 when I put picture to  one NOTE then program was slow, so I delete the file and was slow steal, so I delete a some notes - REORGANIZE - and is great. THANK YOU FOR THE SOFT. helping mi learn and in 05.2024 can be something bad for the world, take care

**Reply by tsujan** [reply]
> I uninstall the soft and compile from git, now IS OK - WORKING EVERYTHING, so.... arch repo has problem?

Oh, you hadn't mentioned that you're an advanced user; otherwise, I'd propose direct compilation. Very good!

And yes, the problem should be in the package that installs FeatherNotes in your second system.

> under LMDE 6 when I put picture to one NOTE then program was slow,...

I have notes with several images, but they're all opened fast. Maybe the images you had were heavy. If your images weren't heavy, please open an issue and attach a sample note that has the problem (as a zipped file).
